packages/EasyActivity/src/ActivityLogEntryFactory.php

```diff
@@ -47,6 +48,7 @@ public function create(string $action, object $object, array $changeSet): ?Activ
             return null;
         }
 
+        $changeSet = $this->getClearedChangeSet($changeSet);
```

Comment:
I think this issue should be solved not in EasyActivity but in EasyDoctrine. Here we rely on events dispatched by EasyDoctrine. And they shouldn't have DateTime objects that weren't changed in event changesets.

This issue is actual for any code relying on EasyDoctrine events. For example, webhooks.

@itorgov what do you think?

Reply:
You know, you are right. Right now `EasyDoctrine` will send an entity updated event when we update entity with the same value in any `date` attribute. It's wrong behavior. And the bad thing that Doctrine will send an update request to a database anyway :-(